Backend/dev/migrations/rider-app/1127-rating.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE atlas_app.rating (
+id character(36) NOT NULL,
+ride_id character varying(36) NOT NULL REFERENCES atlas_app.ride (id),
+rating_value int,
```

Comment:
NOT NULL if it is not null in the actual code

Reply:
feedback_details column is maybe so we can give null